docs/user-guide/ze-install-uninstall-ze.md

```diff
@@ -0,0 +1,48 @@
+# Uninstalling Zowe Explorer
+
+You can uninstall Zowe™ Explorer from Visual Studio Code (VS Code) if you no longer need to use it.
+
+:::info Required roles: systems administrator
+:::
+
+:::warning Important
+The uninstall process does not delete the profiles and credentials that you created when using the product from your computer. To delete the profiles from your computer, delete them before you uninstall Zowe Explorer.
+:::
+
+## Uninstalling the Zowe Explorer VS Code extension
```

Comment:
Zowe Explorer itself cannot be uninstalled if one of the extenders are still present (such as CICS, FTP, etc.). Should we add a clarifying note under this section that a user will not be able to uninstall Zowe Explorer until all Zowe Explorer extensions are uninstalled?